app/src/main/java/unipd/se18/PostProcessingActivity.java

```diff
@@ -0,0 +1,67 @@
+package unipd.se18;
+
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.widget.Button;
+
+import com.yalantis.ucrop.view.OverlayView;
+import com.yalantis.ucrop.view.UCropView;
+
+/**
+ * @author Leonardo Rossi (g2)
+ */
+public class PostProcessingActivity extends AppCompatActivity
+{
+
+    private UCropView cropView;
+    private Button btnConfirmCrop;
+    private MenuItem btnResetChanges;
+    private Uri captureImageUri;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_post_processing);
+
+        //Get intent extra
+        captureImageUri = getIntent().getParcelableExtra("imgUri");
+
+        //Cropview initialization
+        cropView = findViewById(R.id.cropView);
+        cropView.getCropImageView().setImageURI(captureImageUri);
+        cropView.getOverlayView().setShowCropFrame(true);
+        cropView.getOverlayView().setFreestyleCropMode(OverlayView.FREESTYLE_CROP_MODE_ENABLE);
+
+        //UI components initialization
+        btnConfirmCrop = findViewById(R.id.btnConfirmCrop);
+    }
+
+    @Override
```

Comment:
Missing specifications for the method onCreateOptionsMenu

Reply:
This method has not been created by me, since it's clearly an overridden method of the class AppCompatActivity about the menu: https://developer.android.com/guide/topics/ui/menus